src/main/resources/application.properties

```diff
@@ -1,4 +1,15 @@
 # Allow Thymeleaf templates to be reloaded at dev time
 spring.thymeleaf.cache=false
+spring.profiles.active=dev
+
```

Comment:
@Dzianiskulesh Please add here spring.profiles.active=prod  and comment it out.

Reply:
done!